### PR TITLE
Add monitor for kube-controller-manager

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

File: docs/monitors/kubernetes-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+(https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/).
+The monitor queries path `/metrics` by default when no path is configured. The monitor converts
+the Prometheus metric types to SignalFx metric types as described [here](prometheus-exporter.md)
+
+An example configuration that pulls metrics from the HTTP endpoint.
+
+```yaml
+monitors:
+- type: kubernetes-proxy
+  host: localhost
```
Comment (keitwb) [inline docs/monitors/kubernetes-controller-manager.md]: Show it with a discoveryRule.

File: docs/monitors/kubernetes-controller-manager.md [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+Exports Prometheus metrics from the [kube-controller-manager]
+(https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/).
+The monitor queries path `/metrics` by default when no path is configured. The monitor converts
+the Prometheus metric types to SignalFx metric types as described [here](prometheus-exporter.md)
+
+An example configuration that pulls metrics from the HTTP endpoint.
+
+```yaml
+monitors:
+- type: kubernetes-proxy
```
Comment (seonsfx) [inline docs/monitors/kubernetes-controller-manager.md]: Did it mean to be `kubernetes-controller-manager`?

File: docs/monitors/kubernetes-controller-manager.md (hunk 1 above)
Comment (asuresh4) [reply docs/monitors/kubernetes-controller-manager.md]: Updated to use a discoveryRule. This depends on #907

File: docs/monitors/kubernetes-controller-manager.md (hunk 2 above)
Comment (asuresh4) [reply docs/monitors/kubernetes-controller-manager.md]: yes, updated

**Pull request merged**